tests_metricflow/snapshots/test_offset_metrics.py/str/DuckDB/test_offset_window_with_grain_smaller_than_offset__query_output.txt

```diff
@@ -0,0 +1,20 @@
+test_name: test_offset_window_with_grain_smaller_than_offset
```

Comment:
add more test data; this one should not look right

Reply:
test data should ensure there are bookings on the same days for 3 consecutive months